src/python/espressomd/particle_data.pyx

```diff
@@ -368,8 +366,7 @@ cdef class ParticleHandle(object):
             IF MASS == 1:
                 check_type_or_throw_except(
                     _mass, 1, float, "Mass has to be 1 floats")
-                if set_particle_mass(self._id, _mass) == 1:
-                    raise Exception("Set particle position first.")
+                set_particle_mass(self._id, _mass)
```

Comment:
Because I removed the return value from the setter, and forgot it for all the other particles.
I *think* this can only fail if the particle does not actually exist (module some internal errors
that should be handled by asserts), in which case I would throw from the core. The error
message is also misleading.

Reply:
So I guess I'm going to remove the error codes from all the functions? Almost all of them have a hard-coded return value anyhow.